satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -216,6 +216,7 @@ model project (
 
     field name           text
     field description    text      ( updatable )
+	field usage_limit	 int64     ( updatable )
```

Comment:
looks like a whitespace mismatch here. is this using spaces and the other lines are using tabs, or is it the other way around?

Reply:
fixed